content/en/serverless/aws_lambda/logs.md

```diff
@@ -3,14 +3,21 @@ title: Log Collection for AWS Lambda
 kind: documentation
 ---
 
+## Overview
+
+Datadog can collect logs from your AWS Lambda functions in two ways:
+
+- The [Datadog Lambda Extension][2], which collects logs in real time and bypasses CloudWatch.
```

Comment:
I would say "pushes logs to Datadog in real time" vs. collects to be clear about how logs are transmitted to Datadog